front/src/vite-env.d.ts

```diff
@@ -1 +1,9 @@
 /// <reference types="vite/client" />
+
+interface ImportMetaEnv {
+  OSRD_API_URL: string;
+  OSRD_EDITOAST_URL: string;
+  OSRD_KEYCLOAK_REALM: string;
+  OSRD_SENTRY_DSN: string;
+  OSRD_SENTRY_ENVIRONMENT: string;
+}
```

Comment:
What is it used for ?

Reply:
To avoid having `any`s for the environment variables. in this config file for example

![Screenshot_20230323_152410](https://user-images.githubusercontent.com/24652545/227233868-cdbf18e0-cdc8-4bcd-95eb-6ceb223e3bd0.png)